internal/services/services_test.go

```diff
@@ -39,6 +39,7 @@ func buildInjector(templateRenderer template.TemplateRenderer, omnitruckURL stri
 	do.ProvideNamedValue[config.ServiceConfig](injector, "config", config.ServiceConfig{
 		LicenseServiceUrl: "http://license-service",
 		OmnitruckUrl:      omnitruckURL,
+		SupportInfra19:    true,
```

Comment:
we should have a testcase for the false value as well

Reply:
Added some disabled test also